biocypher_metta/adapters/abc_adapter.py

```diff
@@ -50,9 +51,10 @@ def get_nodes(self):
                     if check_genomic_location(self.chr, self.start, self.end, chr, pos, pos):
                         _props = {
                             'chr': chr,
-                            'pos': pos,
+                            'start': pos,
+                            'end': pos,
                             'biochemical_activity': 'DNase I hypersensitive',
-                            'biological_context': row[COL_DICT['cell_type']]
+                            'biological_context': self.tissue_to_ontology_id_map[row[COL_DICT['cell_type']]]
```

Comment:
Since I've generated the `tissue_to_ontology_id_map` from the abc dataset, it already includes entries like `-Roadmap`, so there's no need to split the string further.